internal/mock/cloudsql.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 Google LLC
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     https://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mock
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"math/big"
+	"time"
+)
+
+// CloudSQLInst represents settings for a specific Cloud SQL instance.
+//
+// Use NewCloudSQLInstance to instantiate.
+type CloudSQLInst struct {
```

Comment:
This type exists to make it easier to configure mock responses, right? If that's the case, I'd like to name it to make it clear it's a testing convenience, and not related at all to the production code. Maybe just `FakeInstance` or `TestInstance`?

Reply:
It seems like it's already clear from `mock.CloudSQLInstance`, but okay.